runtime/near-vm-logic/src/gas_counter.rs

```diff
@@ -22,27 +22,98 @@ pub struct GasCounter {
     prepaid_gas: Gas,
     is_view: bool,
     ext_costs_config: ExtCostsConfig,
+
+    /// The cost of one regular operation in Gas.
+    regular_op_cost: Gas,
```

Comment:
Doesn't this approach prevent us from having different per-Wasm op costs?

Reply:
Per wasm op costs are within injected gas meter and tied to this base cost